webpack/assets/javascripts/react_app/components/common/ResourceSwitcher/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Popover,
+  Button,
+  Icon,
+  ListGroup,
+  ListGroupItem,
+  OverlayTrigger,
+} from 'patternfly-react';
+import { ButtonToolbar } from 'react-bootstrap';
+import EllipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import PropTypes from 'prop-types';
+import './ResourceSwitcher.scss';
+
+const ResourceSwitcher = ({ resources }) => {
+  const popover = (
+    <Popover id="popover-contained">
+      <ListGroup className="scrollable-list">
+        {resources.map((item) => {
```

Comment:
what if this is a very long list? should we limit it somehow? did you guys discuss on if we should filter it?

Reply:
yes we should use an autocomplete search bar, as soon as `autocomplete` component will merge, I'll update this. With this iterative approach I think we can merge components faster

Reply:
The original mockups for the katello side include search and pagination of the results.

Reply:
@waldenraines 
I don't like the pagination though, looks odd in a dropdown menu, patternfly design doesn't include it either.

Reply:
@Rohoover thoughts?

Reply:
I included pagination because of the large number of rows. Forcing the user to search for something doesn't allow for browsing. I also did not want to re-introduce anything like infinite scroll since previous customer feedback was disastrous.